resources/views/auth/passwords/change.blade.php

```diff
@@ -0,0 +1,149 @@
+@extends('layouts.password')
+@section('title')
+Change Password
+@endsection
+@section('content')
+<div class="d-flex flex-column" style="min-height: 100vh" id="changePassword">
+    <div class="flex-fill">
+        <div align="center" class="p-5">
+            @php
+            $loginLogo = \ProcessMaker\Models\Setting::getLogin();
+            $isDefault = \ProcessMaker\Models\Setting::loginIsDefault();
+            if ($isDefault) {
+                $class = 'login-logo-default';
+            } else {
+                $class = 'login-logo-custom';
+            }
+            @endphp
+            <img src={{$loginLogo}} alt="{{ config('logo-alt-text', 'ProcessMaker') }}" class="{{ $class }}">
+        </div>
+
+        <div class="row">
+            <div class="col-md-8 offset-md-2 col-lg-6 offset-lg-3">
+                <div class="card card-body p-3">
+                    <form method="PUT" class="form" action="" ref="changePasswordForm">
+                        <div class="text-center pb-4">
+                            <avatar-image size="75" :input-data="{{ $user }}" hide-name="true"></avatar-image>
+                            <h5 class="mt-3">Welcome, <b>{{ $user->fullname }}</b></h5>
```

Comment:
Please translate this string
```suggestion
                            <h5 class="mt-3">{{ __('Welcome, :name', ['name' => $user->fullname]) }}</h5>
```